config.mk

```diff
@@ -1,2 +0,0 @@
-PREFIX = /usr/local
```

Comment:
Are you thinking creating a config.mk file should be part of the install process?